packages/analytics-browser/src/browser-client.ts

```diff
@@ -119,28 +118,14 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
     }
 
     // Add web attribution plugin
-    if (!this.config.attribution?.disabled) {
-      const webAttribution = webAttributionPlugin({
-        excludeReferrers: this.config.attribution?.excludeReferrers,
-        initialEmptyValue: this.config.attribution?.initialEmptyValue,
-        resetSessionOnNewCampaign: this.config.attribution?.resetSessionOnNewCampaign,
-      });
-
-      // For Amplitude-internal functionality
-      // if pluginEnabledOverride === undefined then use plugin default logic
-      // if pluginEnabledOverride === true then track attribution
-      // if pluginEnabledOverride === false then do not track attribution
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      (webAttribution as any).__pluginEnabledOverride =
-        isNewSession || this.config.attribution?.trackNewCampaigns ? undefined : false;
+    if (isAttributionTrackingEnabled(this.config.defaultTracking)) {
+      const attributionTrackingOptions = getAttributionTrackingConfig(this.config);
+      const webAttribution = webAttributionPlugin(attributionTrackingOptions);
       await this.add(webAttribution).promise;
     }
 
     // Add page view plugin
-    const pageViewTrackingOptions = getPageViewTrackingConfig(this.config);
-    pageViewTrackingOptions.eventType = pageViewTrackingOptions.eventType || DEFAULT_PAGE_VIEW_EVENT;
```

Comment:
Do we still support the ability to send a customize event type for page view event?